test-packages/nightly-tests/package.json

```diff
@@ -8,10 +8,12 @@
   "repository": "github:SAP/cloud-sdk-js",
   "scripts": {
     "test": "yarn jest",
-    "check:dependencies": "depcheck ."
+    "check:dependencies": "depcheck . --ignores=@sap-cloud-sdk/generator,@sap-cloud-sdk/openapi-generator"
   },
   "devDependencies": {
     "@sap-cloud-sdk/cli": "^1.37.1",
+    "@sap-cloud-sdk/generator": "^1.37.1",
+    "@sap-cloud-sdk/openapi-generator": "^1.37.1",
```

Comment:
Please correct me, I think the generators should not be used in this package, and only the cli is the target. So we can remove these two dependencies. The `check:dependencies` does not need the ignore then.

Reply:
oh yes. You are right. Unfortunately the cli does not include the openapi client yet. We have a ticket for this.